tests/templates/test_embeddings/test_amplitude.py

```diff
@@ -453,3 +453,47 @@ def test_torch(self, tol, features, pad_with, normalize):
         res2 = circuit2(features, pad_with)
 
         assert qml.math.allclose(res, res2, atol=tol, rtol=0)
+
+
+@pytest.mark.jax
+def test_jacobian_with_and_without_jit_has_same_output_with_high_shots():
```

Comment:
Missing docstring :upside_down_face: 
```suggestion
def test_jacobian_with_and_without_jit_has_same_output_with_high_shots():
    """Test that the jacobian of AmplitudeEmbedding is the same with and without jit"""
```